src/core_plugins/kibana/public/doc/index.html

```diff
@@ -1,47 +1,74 @@
 <div ng-controller="doc" class="app-container">
+  <div class="kuiLocalNav">
+    <div class="kuiLocalNavRow">
+      <div class="kuiLocalNavRow__section">
+        <div class="kuiLocalTitle">
+          Single Entry
```

Comment:
Can we change this to "Single Document", for consistency? From a quick search, I found several instances of us referring to this type of object as a document in the codebase, but none in which we refer to it as an entry.